src/util/draws/draw_helper_functions.jl

```diff
@@ -18,9 +33,24 @@ function rate_to_probability(for_gen::Float64, mttr::Int64)
     return (λ=(μ * for_gen) / (1 - for_gen), μ=μ)
 end
 
-##############################################
-# Initializing Availability
-##############################################
+"""
+initialize_availability!(rng, availability, nexttransition, devices, t_last)
+
+# Arguments
+
+  - `rng::Random.AbstractRNG`: Random number generator
+  - `availability::Vector{Bool}`: Vector of availability status
+  - `nexttransition::Vector{Int}`: Vector of next transition time
+  - `devices::Vector{PSY.Generator}`: Vector of devices
```

Comment:
Vector of devices with outage SupplementalAttribute to be more accurate.

Reply:
Fixed